benchmark/index.ts

```diff
@@ -1,12 +1,11 @@
-'use strict';
 import {URL} from 'url';
-import https = require('https');
+import * as https from 'https';
```

Comment:
Why this change?

Reply:
To prepare for ESM. `= require(…)` is not allowed when compiling to native ESM.